app/assets/stylesheets/style.css

```diff
@@ -233,7 +234,7 @@ ul > li {
   border-radius: var(--border-radius);
   padding: 20px;
   margin: 20px 0;
-  max-width: 300px;
+  width: auto;
```

Comment:
- [ ] Kindly use `width: 100%;` to look similar to the wireframe, as requested by @cdepantha in this [PR](https://github.com/mudasiri/mybloggerapp/pull/9), after applying the changes it will look like this 👇🏽 

![image](https://github.com/mudasiri/mybloggerapp/assets/58642949/44167e4c-aca7-4de1-86e5-9a18c59a948e)

Reply:
@franclobo  I have updated it, please check and approve as I have exhausted my reviews.

Reply:
@franclobo